docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,219 @@
+* Change `SET TIME ZONE LOCAL` to reset to the initial time zone of the client
+  session. ({issue}`15314`)
+* Ensure that the configured number of task retries is not larger than 126. ({issue}`14459`)
+* Fix incorrect rounding of `time(n)` and `time(n) with time zone` values near
+  the top of the range of allowed values. ({issue}`15138`)
+* Fix incorrect results for queries involving window functions without a
+  `PARTITION BY` clause followed by the evaluation of window functions with a
+  `PARTITION BY` and `ORDER BY` clause. {(issue)`15203`)
+* Fix incorrect results when adding or subtracting an `interval` from a
+  `timestamp with time zone`. ({issue}`15103`)
+* Fix potential incorrect results when joining tables on indexed and non-indexed
+  columns at the same time. ({issue}`15334`)
+* Fix potential failure of queries involving `MATCH_RECOGNIZE`. ({issue}`15343`)
+* Fix incorrect recording of `Projection CPU time` in the output of `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`15364`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Fix some `system.metadata` tables improperly showing the names of catalogs
+  which the user cannot access. ({issue}`14000`)
+* Fix `USE` statement improperly disclosing the names of catalogs and schemas
+  which the user cannot access. ({issue}`14208`)
+* Fix improper HTTP redirect after OAuth 2.0 token refresh. ({issue}`15336`)
+
+## Web UI
+
+* Display operator CPU time in the "Stage Performance" tab. ({issue}`15339`)
+
+## JDBC driver
+
+* Return correct values in `NULLABLE` columns of the
+  `DatabaseMetaData.getColumns` result. ({issue}`15214`)
+
+## BigQuery connector
+
+* Add support for using Arrow serialization when reading from BigQuery. This is
+  an experimental feature and can be enabled by setting the
+  `bigquery.experimental.arrow-serialization.enabled` catalog configuration
+  property to `true`. ({issue}`14972`)
+* Fix queries incorrectly executing with the project ID specified in the
+  credentials instead of the project ID specified in the `bigquery.project-id`
+  catalog property. ({issue}`14083`)
+
+## Delta Lake connector
+
+* Add support for views. ({issue}`11609`)
+* Improve performance and reduce storage requirements when running the `vacuum`
+  procedure on S3-compatible storage. ({issue}`15072`)
+* Improve performance of reading Parquet files for `boolean`, `tinyint`,
+  `short`, `int`, `long`, `float`, `double`, `short decimal`, `UUID`, `time`,
+  `decimal`, `varchar`, and `char` data types. The
+  `parquet.optimized-reader.enabled` catalog configuration property can be set
+  to `false` to disable the optimized implementation. ({issue}`14423`, {issue}`14667`)
+* Improve query performance when the `nulls fraction` statistic is not available
+  for some columns. ({issue}`15132`)
+* Improve performance when reading Parquet files. ({issue}`15257`)
```

Comment:
I believe there's a follow up PR to address an issue introduced by this one. We should link to both.

Reply:
That one is https://github.com/trinodb/trino/pull/15474
It can be landed after CI passes

Reply:
FYI https://github.com/trinodb/trino/pull/15474 is merged now